src/stylesheets/components/_hero.scss

```diff
@@ -2,36 +2,36 @@
 // ==========================
 
 .usa-hero {
-  @include u-padding-y($site-margins);
+  @include u-padding-y(units($theme-site-margins));
   background-image: url('#{$theme-image-path}/hero.png');
   background-position: center;
   background-size: cover;
 
   .usa-button {
-    color: $color-white;
+    color: color('white');
   }
 }
 
 .usa-section-dark {
   .usa-hero-link {
-    color: $color-primary-alt;
+    color: color('accent-cool');
   }
 }
 
 .usa-hero-callout {
-  background-color: $color-primary-darkest;
-  padding: 3rem;
+  background-color: color('primary-darker');
+  padding: units(4);
 
-  @include media($small-screen) {
+  @include at-media('mobile-lg') {
     max-width: 30rem;
   }
 
   > *:first-child {
-    @include margin(0 null $spacing-medium null);
+    @include margin(0 null units(2) null);
```

Comment:
We are going to deprecate using Bourbon I believe and this is a Bourbon mixin, so this should probably be written in a 2.0 margin way.